pkg/gofr/container/datasources.go

```diff
@@ -266,3 +266,63 @@ type KVStoreProvider interface {
 
 	provider
 }
+
+// Dgraph defines the methods for interacting with a Dgraph database.
+type Dgraph interface {
+	// Query executes a read-only query in the Dgraph database and returns the result.
+	// Parameters:
+	// - ctx: The context for the query, used for controlling timeouts, cancellation, etc.
+	// - query: The Dgraph query string in GraphQL+- format.
+	// Returns:
+	// - interface{}: The result of the query, usually of type *api.Response.
+	// - error: An error if the query execution fails.
+	Query(ctx context.Context, query string) (interface{}, error)
```

Comment:
```suggestion
	Query(ctx context.Context, query string) (any, error)
```

Here and everywhere

Reply:
Sure will replace `interface{}` with `any` in the entire file